src/webview/repl-console.ts

```diff
@@ -323,16 +359,20 @@ export class ReplConsole {
 
     commands = {
         "raise-sexp": () => {
-            this.readline.withUndo(() => {
-                paredit.raiseSexp(this.readline);
-                this.readline.repaint();
-            });
+            if (this.isKeyMay([ReplPareditKeyMap.ORIGINAL, ReplPareditKeyMap.STRICT])) {
```

Comment:
Why do we check this here? And, why only on some commands and not on others?

Reply:
I tried to mimic the when clauses in the package.json. I put the check around all commands where paredit is involved. I have no idea if this makes sense.

Reply:
I don't think it makes sense. The when clause is in effect also in the REPL window. Try it in 2.0.49, setting the keymap to `none` disables Paredit shortcuts in the REPL window.

Reply:
Okay what should I do then? Remove the check everywhere except backspace and delete?

Reply:
Or only at 'raise-sexp'? Why do the shortcuts from package.json work in the repl?